community_server/src/Model/Table/TransactionsTable.php

```diff
@@ -168,7 +168,11 @@ public function listTransactionsHumanReadable($stateUserTransactions, array $use
         if($decay_start_transaction->count()) {
             $decay_start_transaction_id = $decay_start_transaction->first()->id;
         }
-        $decay_start_time = $stateBalancesTable->getDecayStartDateCached()->getTimestamp();
+        $decay_start_date = $stateBalancesTable->getDecayStartDateCached();
+        $decay_start_time = 0;
+        if($decay_start_date) {
+            $decay_start_time = $decay_start_date->getTimestamp();
+        }
```

Comment:
Once again. This is another PR. Please try to have proper PRs.

Reply:
I have like yesterday proposed put that in another branch and pr (#640) and merge it with that branch so that I can test my changes properly.